config/routes.rb

```diff
@@ -2,9 +2,10 @@
   scope "(:locale)", locale: /en|vi/ do
     root "static_pages#home"
 
-    get "static_pages/home"
-    get "static_pages/help"
-    get "static_pages/about"
-    get "static_pages/contact"
+    get "/home", to: "static_pages#home"
+    get "/help", to: "static_pages#help"
+    get "/about", to: "static_pages#about"
+    get "/contact", to: "static_pages#contact"
+    get "/signup", to: "users#new"
```

Comment:
`to` để làm gì e ?

Reply:
to để tạo route riêng /help thay cho static_page/help